tests/test_visualization.py

```diff
@@ -6,7 +6,10 @@
 from mesa.time import SimultaneousActivation
 from mesa.visualization.ModularVisualization import ModularServer
 from mesa.visualization.modules import CanvasGrid, TextElement
-from mesa.visualization.UserParam import UserSettableParameter
+from mesa.visualization.UserParam import (
+    NumberInput,
+    Slider,
+)
 from tests.test_batchrunner import MockAgent
```

Comment:
I think I am missing some here on the conflict. If we are importing `MockAgent` shouldn't it be `tests.test_batch_run`?

Reply:
That was the code from before `BatchRunner` was removed.